src/tracing/decorators/span.ts

```diff
@@ -1,6 +1,12 @@
-import { context, trace } from '@opentelemetry/api';
+import { context, SpanStatusCode, trace } from '@opentelemetry/api';
 
-export function Span(name?: string) {
+type Param = {
+  name?: string;
+  setStatus?: boolean;
+};
+
+export function Span(param: Param = {}) {
+  const { name, setStatus = true } = param;
```

Comment:
Better stay close to the original signature as much as possible

```suggestion
export function Span(name?: string, setStatus = true) {
```